### PR TITLE
add dystopia

### PR DESCRIPTION
**Pull request by Frisasky**
# Description

Add dystopia server, based on dystopia linux server guide
https://github.com/dysemjay/dys-server-guide/

Fixes #2245 

## Type of change

* [ ] Bug fix (change which fixes an issue)
* [ ] New feature (change which adds functionality)
* [x] New Server (new server added)
* [ ] Refactor (restructures existing code)
* [x] This change requires a documentation update

## Checklist

* [x] My code follows the style guidelines of this project
* [x] This pull request links to an issue
* [x] This pull request uses the `develop` branch as its base 
* [ ] I have performed a self-review of my own code
* [ ] I have squashed commits
* [ ] I have commented my code, particularly in hard to understand areas
* [x] I have made corresponding changes to the documentation if required

### CODE REVIEW

**Comment by dgibbs64** [comment]
Thanks for this!

**Comment by dgibbs64** [comment]
There is an issue with that that Source SDK Base 2013 Dedicated Server https://steamdb.info/app/244310/ is required to be installed first. Meaning two installations are required. There isn't currently an option for this is LinuxGSM. But is very possible to do. I reccomend an if statement to detect game that require 244310

**Comment by Frisasky** [comment]
yeah......... i saw theres goldeneye so i thought lgsm would take care of sourcesdk things, dunno its a bit more complicate

File: lgsm/config-default/config-lgsm/dysserver/_default.cfg [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+ip="0.0.0.0"
+port="27015"
+clientport="27005"
+sourcetvport="27020"
+defaultmap="dys_broadcast"
+maxplayers="16"
+
+## Server Start Command | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
+fn_parms(){
+parms="-game ${serverfiles}/dystopia -strictportbind -ip ${ip} -port ${port} +clientport ${clientport} +tv_port ${sourcetvport} +map ${defaultmap} +servercfgfile ${servercfg} -maxplayers ${maxplayers}"
```
Comment (dgibbs64) [inline lgsm/config-default/config-lgsm/dysserver/_default.cfg]: Add double quotes here "${serverfiles}/dystopia" should prevent issues if someone decides to have a dir with a space in it.

**Comment by dgibbs64** [comment]
Commit message

Co-authored-by: digimoncn@gmail.com

feat(newserver): Dystopia

**Pull request merged**